GitUI/UserControls/RevisionGrid.cs

```diff
@@ -328,14 +328,13 @@ private void Loading_Paint(object sender, PaintEventArgs e)
         [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
         public Font NormalFont
         {
-            get { return _normalFont; }
+            get => _normalFont;
```

Comment:
for consistency, please keep it in the original syntax.
if both `get` and `set` can be inlined, we inline them